bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/LedgerUnderreplicationManager.java

```diff
@@ -240,6 +240,7 @@ boolean initializeLostBookieRecoveryDelay(int lostBookieRecoveryDelay)
      * @param cb
      * @throws ReplicationException.UnavailableException
      */
+    @Deprecated
```

Comment:
We'd better add a default empty implementation for the interface, otherwise, all the implementation classes need to implement this method

Reply:
Make sense.